zoo-css-challenge/style.css

```diff
@@ -166,18 +171,27 @@ footer {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 25px;
+  button:hover 
```

Comment:
```suggestion
```
`button:hover` is a CSS selector and cannot be inside another CSS selector — here it is inside the `.education-container` selector. It needs to be moved outside the `{` `}` block for it to work